packages/arb-token-bridge-ui/src/ui-driver/UiDriverCctp.test.ts

```diff
@@ -11,17 +31,22 @@ it(`
   user actions:
     1. user rejects "confirm_cctp_deposit" dialog
 `, async () => {
-  const generator = stepGeneratorForCctp({
+  const context: UiDriverContext = {
     isDepositMode: true,
     isSmartContractWallet: false
-  })
+  } as UiDriverContext
```

Comment:
why not add `amountBigNumber`, `walletAddress`, and `transferStarter` instead of casting it here?

Reply:
they are not used in this test